src/components/DialogContentContainer/DialogContentContainer.tsx

```diff
@@ -1,12 +1,21 @@
 import { SIZES } from "../../constants/sizes";
 import React, { useRef, forwardRef } from "react";
-import PropTypes from "prop-types";
 import cx from "classnames";
 import useMergeRefs from "../../hooks/useMergeRefs";
 import { BEMClass } from "../../helpers/bem-helper";
-
+import VibeComponentProps from "src/types/VibeComponentProps";
 import "./DialogContentContainer.scss";
 
+interface DialogContentContainerProps extends VibeComponentProps {
+  children: React.ReactNode;
```

Comment:
please mark all the props which are not required as optional

Reply:
I have one doubt...Is children prop optional?